src/clj/rems/css/styles.clj

```diff
@@ -844,6 +845,12 @@
                        :padding "0.2rem"
                        :border-radius "0.25rem"}]]
 
+   ;; special case for previous applications
+   [:#previous-applications-except-current [:.resource {:overflow :hidden
+                                                        :text-overflow :ellipsis
+                                                        :white-space :nowrap
+                                                        :max-width "10em"}]]
```

Comment:
should we use `garden.units` everywhere for consistency? or switch to writing units as here

Reply:
I don't have a clear answer at the moment. I used to write using the units, but mostly it seems like a waste of time. We can consider unifying how we write them when we restyle the UI.